src/nft/storage.test.ts

```diff
@@ -52,6 +65,10 @@ describe("Testing getting NFT data", (): void => {
     const maybeNft = await getNftData(TEST_DATA.nftId)
     expect(maybeNft != null).toBe(true)
   })
+  it("Should return secret offchain data when the NFT ID exists and is a Secret NFT", async () => {
+    const actual = await getSecretNftOffchainData(96)
```

Comment:
We cannot use an hardcoded `nftId` here, otherwise it will be network dependent.
To test NFT related storage getters, we create a specific NFT in the `beforeAll` function (file's top) and store its ID in the `TEST_DATA` constant.

Reply:
Fixed